src/main.rs

```diff
@@ -173,6 +175,23 @@ fn validate_byte_size(s: String) -> Result<(), String> {
         |_| format!("{:?} is not a byte size", s))
 }
 
+// Create a regular expression from a file pattern:
+// Change
+// 1. "." to "\\."
+// 2. "%" to "."
```

Comment:
Wouldn't this be a `?` for a single char? I'm not familiar with `%` being used in file patterns.

Reply:
Yes, you are right.